src/frontend/src/components/ReactHookEditableList.tsx

```diff
@@ -1,4 +1,4 @@
-import { Grid, Button, IconButton, TextField } from '@mui/material';
+import { Grid, Button, IconButton, TextField, Typography } from '@mui/material';
 import DeleteIcon from '@mui/icons-material/Delete';
```

Comment:
IMO we should have consistency across files and within imports in the same file...